config/seo_meta.json

```diff
@@ -6,14 +6,13 @@
     "title": "ACME Storefront | Powered by Next.js Commerce",
     "description": "Next.js Commerce - https://www.nextjs.org/commerce",
     "type": "website",
-    "locale": "en_IE",
-    "url": "https://nextjs.org/commerce",
+    "url": "https://demo.vercel.store/",
     "site_name": "Next.js Commerce",
     "images": [
       {
-        "url": "/card.png",
-        "width": 800,
-        "height": 600,
+        "url": "https://demo.vercel.store/card.png",
```

Comment:
This points only to the bigcommerce version, is that expected?

Reply:
good point, I added new ENV var